operations/mimir-tests/test-multi-zone.jsonnet

```diff
@@ -19,6 +19,14 @@ mimir {
 
     multi_zone_store_gateway_enabled: true,
     multi_zone_store_gateway_replicas: 3,
+
+    local availabilityZones = ['us-east-2a', 'us-east-2b'],
+    multi_zone_distributor_enabled: true,
+    multi_zone_distributor_availability_zones: availabilityZones,
+
+    autoscaling_distributor_enabled: true,
```

Comment:
Note to reviewers:
I have added autoscaling here in order to test autoscaling combined with multi-AZ, because we otherwise don't test it anywhere else.

Reply:
Can we create a copy of this file instead? This file is imported by many other files (eg. ones to show migration steps), but we don't want to change them all.

Reply:
I see what you mean, it's just that this file is used as a base for our migration procedure to ingest-storage, and nobody is running with multi-zone distributors today.

Reply:
As discussed in a Slack thread, we decided to move distributor-related part in `test-multi-zone-distributor.jsonnet`.